webhook_server/libs/config.py

```
@@ -44,8 +44,8 @@ def repository_data(self) -> dict[str, Any]:
 
     def repository_local_data(self, github_api: github.Github, repository_full_name: str) -> dict[str, Any]:
         if self.repository and repository_full_name:
-            # Import here to avoid cyclic imports
-            from webhook_server.utils.helpers import get_github_repo_api
+            # Import here to avoid circular imports
+            from webhook_server.utils.helpers import get_github_repo_api  # noqa: PLC0415
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)